Tasks/ANT/package.json

```diff
@@ -1,12 +1,14 @@
 {
-  "name": "Ant",
+  "name": "Agent.Tasks.Ant",
+  "version": "1.0.52",
```

Comment:
no need for duplicating task version here.  the package.json is for getting the dependencies